builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -41,7 +42,10 @@
     private boolean keycloakDisabled;
 
     @Inject
-    private KeycloakUserChecker userChecker;
+    private KeycloakTokenStore         kcTokenStore;
```

Comment:
Personally, I find it more readable than useless constructor and parameters in constructor when it's actually doing nothing else then just setting the value to the field. Using injection or mock framework is not a problem to me.